### DISCUSSION TITLE
CRSF: extra 2 bytes trailing DEVICE_PING packet?

### DISCUSSION THREAD

**Discussion by Modiug** [ExpressLRS/ExpressLRS#1910] [Q&A] [answered]
I am trying to get a Jumper AION Nano TX to work with a BetaFPV LiteRadio 3. The module is not recognized.
A logic analyzer revealed: The LiteRadio 3 sends two extra bytes end with that overwrites the first byte of the reply from the Jumper AION.

What irritates me, is that the example [here](https://github.com/ExpressLRS/ExpressLRS/wiki/CRSF-Protocol#example-loading) also shows a DEVICE_PING packet [EE 04 28 00 EA 54 2B C0] with two additional bytes: 2B C0.
So, is this a known phenomenon?
How can I debug this further? I would like to delay the reply to the DEVICE_PING to see if this would help.

**Comment by Modiug** [comment]
OK, I get this behavior with ELRS Version 3.0.1 on the Jumper AION Nano TX, but not with the current [master](https://github.com/ExpressLRS/ExpressLRS/commit/f71b550413a7a7b2e30ee7e622ca49c8faa22c4b).

However it still does not work. LiteRadio 3 keeps on sending DEVICE_PING, the Jumper AION replies with DEVICE_INFO. I made myself a simple decoder and it looks like this:
```
CRSF_TRANSMITTER: DEVICE_PING: dst=BROADCAST, src=RADIO_TRANSMITTER
extra: 93 17
CRSF_TRANSMITTER: DEVICE_PING: dst=BROADCAST, src=RADIO_TRANSMITTER
extra: 93 17 00
CRSF_TRANSMITTER: DEVICE_PING: dst=BROADCAST, src=RADIO_TRANSMITTER
extra: 93 17
RADIO_TRANSMITTER: RADIO_ID: ea ee 10 00 00 9c 40 00 00 02 44
CRSF_TRANSMITTER: DEVICE_PING: dst=BROADCAST, src=RADIO_TRANSMITTER
extra: 93 17
RADIO_TRANSMITTER: DEVICE_INFO: dst=RADIO_TRANSMITTER, src=CRSF_TRANSMITTER, disp_name='AION 2G4NANO TX', ser='ELRS', hw_ver=[00 00 00 00], sw_ver=[00 03 00 00], nr_cfg_params=21, param_proto_ver=0
CRSF_TRANSMITTER: DEVICE_PING: dst=BROADCAST, src=RADIO_TRANSMITTER
extra: 93 17
RADIO_TRANSMITTER: DEVICE_INFO: dst=RADIO_TRANSMITTER, src=CRSF_TRANSMITTER, disp_name='AION 2G4NANO TX', ser='ELRS', hw_ver=[00 00 00 00], sw_ver=[00 03 00 00], nr_cfg_params=21, param_proto_ver=0
CRSF_TRANSMITTER: DEVICE_PING: dst=BROADCAST, src=RADIO_TRANSMITTER
extra: 93 17
RADIO_TRANSMITTER: DEVICE_INFO: dst=RADIO_TRANSMITTER, src=CRSF_TRANSMITTER, disp_name='AION 2G4NANO TX', ser='ELRS', hw_ver=[00 00 00 00], sw_ver=[00 03 00 00], nr_cfg_params=21, param_proto_ver=0
CRSF_TRANSMITTER: DEVICE_PING: dst=BROADCAST, src=RADIO_TRANSMITTER
extra: 93 17
```
It seems like the DEVICE_INFO packet is ignored by the LiteRadio 3.

Well, I guess, I give up here as I cannot look into BetaFPV's LiteRadio 3 firmware.

**Comment by Modiug** [comment, marked as answer]
From what I understood on Discord: LiteRadio 3 (non-pro!) does not work with ELRS TX modules having ELRS versions > 2.0.1. Bummer!

**Comment by Ncerzzk** [comment]
I don't know how the firmware do in LR3, while you can flash the firmware of LR3 back to 1.01, and then ELRS3.0 could be used with it.

This is totally a bug of LR3, while after I report to betafpv, there is no indication that they will fix it.

**Reply by Modiug** [reply]
Thanks for the information. Good to know.
But meanwhile I bought a radio running EdgeTX with internal ELRS transmitter. That worked painlessly.
